src/bip340.rs

```diff
@@ -409,10 +407,8 @@ mod tests {
             let verify_result = verify_single(pubkey, test_vec_signature, &record.message);
             match record.verification_result.as_str() {
                 "TRUE" => {
-                    verify_result.expect(&format!(
-                        "verification should pass for signature {} - {}",
-                        &record.signature, record.comment,
-                    ));
+                    verify_result.unwrap_or_else(|_| panic!("verification should pass for signature {} - {}",
+                        &record.signature, record.comment));
```

Comment:
I think you may need to run `cargo fmt` on this PR

Reply:
damn i really need to get off my ass, and set up github actions CI

Reply:
Done